pkg/cloud/awsprovider.go

```diff
@@ -1407,18 +1418,10 @@ func (a *AWS) GetAddresses() ([]byte, error) {
 			defer errors.HandlePanic()
 
 			// Query for first page of volume results
-			resp, err := a.getAddressesForRegion(region)
+			resp, err := aws.getAddressesForRegion(context.TODO(), region)
 			if err != nil {
-				if aerr, ok := err.(awserr.Error); ok {
-					switch aerr.Code() {
-					default:
-						errorCh <- aerr
-					}
-					return
-				} else {
-					errorCh <- err
-					return
-				}
+				errorCh <- err
+				return
```

Comment:
awserr casting did not make a difference here

Reply:
Yikes. Yeah, good fix!